include/SolutionOptions.h

```diff
@@ -204,7 +203,7 @@ class SolutionOptions
   std::map<std::string, std::string> inputVarFromFileMap_;
 
   // mesh motion
-  std::map<std::string, MeshMotionInfo *> meshMotionInfoMap_;
+  YAML::Node meshMotionNode_;
```

Comment:
@sayerhs I need the realm to initialize `meshMotionAlg_` and other mesh motion related classes therein. `SolutionOptions` unfortunately does not own an instance of `Realm`, so I am not sure what would be the best way to go about this. One option I can think of -- Create  `meshMotionAlg_` in `SolutionOptions` but then set the relevant member variables (meta_data, bulk_data) in `Realm::load()`. Would that be cleaner ?

Reply:
How about making `mesh_motion` section in input file a part of `realm:` like `actuator:`, `abl_forcing:`, etc.? Since Realm owns the `meshMotionAlg_` it would remove SolutionOptions from being involved at all.

Reply:
I have opened an issue for this #78. Will start working on it soon.

Reply:
This issue has been resolved in https://github.com/Exawind/nalu-wind/pull/77/commits/43f9b07074f7b3a868e2aea6b03b4bf75e1f1654 and https://github.com/Exawind/nalu-wind/pull/77/commits/d168accf15240cf1d7275bb1a3f4c10f6ca1b868